### DISCUSSION TITLE
Will there be a more customisable lazer skinning experience?

### DISCUSSION THREAD

**Discussion by 7deok7** [ppy/osu#24884] [Q&A] [answered]
Currently creating a skin from scratch is quite hard.

For anyone who doesn't know how that's done, this is an example made by peppy: https://github.com/ppy/osu/discussions/24241#discussioncomment-7029783

I just wish there would be more you can do with how lazer looks. For example, changing how the song select looks. This was one of the things that made stable skins stand out from one another, beautiful, well designed song selections. There was a lot of variety, every song select looked (mostly) different from skin to skin. Now it's just the same, and it looks quite boring to me. Same things goes for ranking panel, pause overlay, etc.

Will this ever be a feature, and if not, why? I just really enjoy skinning, and one of my favorite parts about skinning is the GUIs.

**Comment by peppy** [comment, marked as answer]
Yes, it's planned.

**Reply by 7deok7** [reply]
Thank you very much <3
